projects/toggle/src/lib/directives/toggle-group.directive.ts

```diff
@@ -69,7 +69,7 @@ export class KcToggleGroupDirective<T> implements OnInit, ControlValueAccessor {
   select(obj: T) {
     this._values.toggle(obj, obj);
 
-    this._onChange(obj);
+    this._onChange(this._values.selected!);
```

Comment:
In case of `multiple = true` we want FormControl to be updated with all values selected, not just  last selected (obj)